### ISSUE TITLE
When launching non-emulator Actions, error message displays.

### ISSUE THREAD

**Issue by TheT113** [Nyarlathotep7777/archive-unpacker#1]
When using any Action (ie to launch an exe, to launch a Steam game, or any other Action) the error message "Object reference not set to an instance of an object." is displayed. Clicking OK will then proceed with launching it as normal.

This only occurs while this add-on is installed.

**Comment by Nyarlathotep7777** [comment]
Thanks for the feedback, fixed in v1.0.3, tested launching a Steam game. Please let me know if it appears again.

**Comment by Nyarlathotep7777** [comment]
Additional fix in v1.0.4

**Comment by TheT113** [comment]
It seems fixed for Steam games, but still occurs when launching an .exe to Playnite, or launching a Link-type play action (an Action that loads a webpage, like for quickly launching Browser-based games). Possibly other types too, but these are the ones I noticed it with. (I just tested it with Epic games and Minecraft (in Xbox library) and no issues there; so it seems to be fixed for any Libraries, but not for imported stuff).

Thank you for the quick fix though, I appreciate this extension!
